packages/common/src/auditlogs/auditlogs.js

```diff
@@ -0,0 +1,265 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { makeStyles } from 'tss-react/mui';
+
+import EnterpriseNotification, { DefaultUpgradeNotification } from '@northern.tech/common-ui/enterpriseNotification';
+import { HELPTOOLTIPS, MenderHelpTooltip } from '@northern.tech/helptips/helptooltips';
+import { AUDIT_LOGS_TYPES, BEGINNING_OF_TIME, BENEFITS, SORTING_OPTIONS, SP_AUDIT_LOGS_TYPES, TIMEOUTS } from '@northern.tech/store/constants';
+import {
+  getAuditLog,
+  getAuditLogEntry,
+  getAuditLogSelectionState,
+  getCurrentSession,
+  getGroupNames,
+  getIsServiceProvider,
+  getTenantCapabilities,
+  getUserCapabilities
+} from '@northern.tech/store/selectors';
+import { getAuditLogs, getAuditLogsCsvLink, getUserList, setAuditlogsState } from '@northern.tech/store/thunks';
+import { createDownload, getISOStringBoundaries } from '@northern.tech/utils/helpers';
+import { useLocationParams } from '@northern.tech/utils/liststatehook';
+import dayjs from 'dayjs';
+
+import historyImage from '../../../../assets/img/history.png';
```

Comment:
I'm not sure how this will behave for the resulting bundle, but I guess we'll see once it's released 🤞